### DISCUSSION TITLE
OBS -> NDI Audio

### DISCUSSION THREAD

**Discussion by ReidHuntWill** [DistroAV/DistroAV#1166] [Q&A - Support]
Hello, 

We recently have been using a setup with NDI (distro-av) that would stream from a computer for our meeting to another pc to stream/broadcast on YouTube. We had been running fine (other than audio skipping, this could be a problem with running OBSBOT tailair over wifi and streaming via a direct ethernet connection. (if you've heard of this or seen issues with this that would be great to hear about too!)) However, the current problem is that we had an update to the C++ code pop up in OBS this morning, once we got all systems booted we could not stream any audio to our stream PC (video was working fine, just a little choppy). We first suspected the PCs had an error with the stream, so we rebooted the obs on both systems (this works sometimes) but it didn't for this instance. After we tried streaming on another computer, we got some audio but it was just blips of 1-2 seconds. We then updated all drivers and made sure OBS was up to date. We finally tried rebooting all systems to see if the error would correct itself, yet we still can't seem to get audio to stream just video. Currently, we have all devices registered on OBS 31, could this be an issue with the recent update and NDI not sharing audio streams? I tried to reset the NDI connection as well to see if I could recast it to the same PC but no luck in this situation either. If you have any suggestions or ideas we would love to hear them. If we need to drop OBS back down to 30 we can but wanted to see what solutions you all might have for us. Thank you! (this is great software that helped us tremendously so we appreciate the creators and developers!)

**Comment by Ross902** [comment]
Hi Reid   The same thing is happening to me.  Before Christmas, I was able to stream from OBS to another computer with audio (Although, the audio was at a low level,  I boosted it in NDI Monitor. )   Today no sound, but the video works.  

I do hope a solution comes up soon.    Any other suggestions for making a OBS into NDI?

**Comment by Trouffman** [comment]
to both of you please open an issue with your OBS log file and all required infos. we cannot help without log file and understand you environment.
